packages/eas-cli/src/submissions/utils/errors.ts

```diff
@@ -28,14 +32,30 @@ const SubmissionErrorMessages: Record<SubmissionErrorCode, string> = {
   [SubmissionErrorCode.ANDROID_MISSING_PRIVACY_POLICY]:
     'The app has permissions that require a privacy policy set for the app.\n' +
     `${learnMore('https://expo.fyi/missing-privacy-policy')}.`,
+  [SubmissionErrorCode.IOS_OLD_VERSION_CODE_ERROR]:
+    "You've already submitted this version of the app.\n" +
+    'Versions are identified by Build Numbers (expo.ios.buildNumber in app.json).\n' +
```

Comment:
maybe we can link to a FYI page here instead, so it's easier to update and provide information about how to do this in managed and bare contexts

Reply:
Done, however I haven't changed this message to reflect the already-existing Android counterpart:
https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/submissions/utils/errors.ts#L23-L27

Reply:
ah, we should follow up and change that on the android side. i think it's because we wrote the android side prior to eas-cli existing